src/TodoItem.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Button from './Button';
+
+function TodoItem({ item, onDoneButtonClick }) {
```

Comment:
```suggestion
function TodoItem({ item: {todo, id}, onDoneButtonClick }) {
```
이렇게 한번 더 destructuring을 할 수 있을 것 같네요 :)